src/app/snapp_test_transaction/README.md

```diff
@@ -0,0 +1,702 @@
+  update-permissions              Generate a snapp transaction that updates the
+                                  permissions of a snapp account
+  test-snapp-with-genesis-ledger  Generate a trivial snapp transaction and
+                                  genesis ledger with verification key for
+                                  testing
+  version                         print version information
+  help                            explain a given subcommand (perhaps
+                                  recursively)
+
+```
+
+### Example usage
+
+#### 1. Create a snapp account / Deploy the test smart contract
+
+`create-snapp-account` command takes the following input to create a snapp account and deploy the test smart contract. 
+
+```shell
+$mina-snapp-test-transaction create-snapp-account -help
+Generate a snapp transaction that creates a snapp account
+
+  mina-snapp-test-transaction create-snapp-account 
+
+=== flags ===
+
+  --fee-payer-key KEYFILE      Private key file for the fee payer of the
+                               transaction (should already be in the ledger)
+  --nonce NN                   Nonce of the fee payer account
+  --receiver-amount NN         Receiver amount in Mina
+  --snapp-account-key KEYFILE  Private key file to create a new snapp account
+  [--debug]                    Debug mode, generates transaction snark
+  [--fee FEE]                  Amount you are willing to pay to process the
+                               transaction (default: 1) (minimum: 0.003)
+  [--memo STRING]              Memo accompanying the transaction
+  [-help]                      print this help text and exit
+                               (alias: -?)
+```
+
+For example:
+
+```shell
+$mina-snapp-test-transaction create-snapp-account --fee-payer-key my-fee-payer --nonce 0 --receiver-amount 2 --snapp-account-key my-snapp-key
+```
+
+generates the following graphql object- a snapp transaction as input to the sendSnapp mutation. A snapp transaction is basically a list of parties where each [party](https://o1-labs.github.io/snapps-txns-reference-impl/target/doc/snapps_txn_reference_impl/party/index.html) is an update performed on an account.
+
+The snapp transaction here has three parties-
+
+1. the fee payer party which specifies who pays the transaction fees and how much
+2. A party that pays the account creation fee to create the new snapp snapp account which in this case is the same as the fee payer
+3. A party to create a new snapp account, set its verification key associated with the test smart contract, and update `editState` and `editSequenceState` permissions to use proofs as [authorization](https://o1-labs.github.io/snapps-txns-reference-impl/target/doc/snapps_txn_reference_impl/party/enum.PartyAuthorization.html).
+
+The authorization used in each of the parties here is a signature of the respective accounts i.e., the updates on these accounts are authorized as per the accounts' permissions.
+
+```json
```

Comment:
the input isn't JSON, in fact, field names are not double-quoted

I guess just remove the language indicator `json`